contracts/forwarding/IForwarderV2.sol

```diff
@@ -0,0 +1,13 @@
+/*
+ * SPDX-License-Identifier:    MIT
+ */
+
+pragma solidity ^0.4.24;
+
+import "./IForwarderPayable.sol";
+
+
+// TODO: Cannot inherit interfaces in Solidity 0.4.24
+contract IForwarderV2 is IForwarderPayable {
+    function forward(bytes evmCallScript, bytes context) external payable;
```

Comment:
We cannot be sure that this `context` works with more than one step in the forwarding chain, right? What would be an example of a use case for this `context` field?